tooling/test/sugar.go

```diff
@@ -8,14 +8,15 @@ import (
 )
 
 type RequestBuilder struct {
-	Method_               string
-	Path_                 string
-	URL_                  string
-	Proxy_                string
-	UseProxyTunnel        bool
-	Headers_              map[string]string
-	DoNotFollowRedirects_ bool
-	Query_                url.Values
+	Method_               string            `json:"method,omitempty"`
+	Path_                 string            `json:"path,omitempty"`
+	URL_                  string            `json:"url,omitempty"`
+	Proxy_                string            `json:"proxy,omitempty"`
+	UseProxyTunnel_       bool              `json:"useProxyTunnel,omitempty"`
+	Headers_              map[string]string `json:"headers,omitempty"`
+	DoNotFollowRedirects_ bool              `json:"doNotFollowRedirects,omitempty"`
+	Query_                url.Values        `json:"query,omitempty"`
+	Body_ 							  []byte            `json:"body,omitempty"`
```

Comment:
weird formatting here

Reply:
Oh, i might have forgotten to go fmt it after the merges.